environment.yml

```
@@ -34,8 +34,8 @@ dependencies:
 - pyyaml~=5.1.2
 - reentry~=1.3
 - simplejson~=3.16
-- sqlalchemy-utils~=0.36.0
-- sqlalchemy~=1.3.10
+- sqlalchemy-utils~=0.37.0
```

I'm afraid we probably have to pin sqlalchemy-utils since the project does not follow any specified versioning scheme (https://github.com/kvesteri/sqlalchemy-utils/issues/438).
```suggestion
- sqlalchemy-utils==0.37.0
```